pkgs/development/libraries/libid3tag/default.nix

```diff
@@ -29,6 +20,16 @@ stdenv.mkDerivation rec {
     )
   '';
 
+  strictDeps = true;
+
+  setOutputFlags = false;
+
+  outputs = [ "out" "dev" ];
```

Comment:
While we redorder things: outputs normally goes between version and src.

Reply:
I thought 'outputs' was configure. But honestly, I have no idea really. A bit odd being between `version` and `src`. What is the logic? (I'm trying to make sense of it.)

Reply:
Hmmm, good question. I don't know.